Assets/Scripts/CarController.cs

```diff
@@ -21,6 +21,7 @@ public class CarController : MonoBehaviour
     public float maxSteerAngle = 30;
     public float motorForce = 50;
     public float maxAngleChangePerSecond = 10;
+    public float velocity;
```

Comment:
Should have [HideInInspector] annotation

Reply:
Done! Had totally forgot my vector math.